integrations/tensorflow/e2e/keras/BUILD

```
@@ -162,8 +162,8 @@ iree_e2e_test_suite(
 )
 
 iree_vision_test_suite(
-    name = "medium_cifar10_internal_tests",
-    size = "medium",
+    name = "long_cifar10_internal_tests",
+    size = "long",
```

s/long/large/

```suggestion
    name = "large_cifar10_internal_tests",
    size = "large",
```

https://docs.bazel.build/versions/master/be/common-definitions.html#test.size